Prototype for Profile page.

eselenic opened the pull request:
Some changes were from Week 0. Still working on designing page and will add more tests.

eselenic committed:
Prototype for Profile page.

Rafer45 commented on src/test/java/codeu/controller/ProfileServletTest.java:
```diff
@@ -0,0 +1,40 @@
+	    Mockito.verify(mockResponse).sendRedirect("/login");
+
+	  }
+
```
Looks good! Could you add a test for when the user is not null?

EDIT: I just saw your message on adding more tests; understood!

Rafer45 commented on src/main/java/codeu/controller/ProfileServlet.java:
```diff
@@ -0,0 +1,51 @@
+/** This servlet class is responsible the profile pages*/
+public class ProfileServlet extends HttpServlet{
+	
+	  /** From my perspective this variable is used to keep track of who is looking at
```
The indentation is a little off from the one in the standard -- instead of tabs, there should be two spaces.

dgalbraith33 replied:
Ditto on this. I'll try to get us set up with a linter soon that will catch formatting issues automatically. :smile:

jcolladokuri replied:
https://www.10bestdesign.com/dirtymarkup/js/

This websites has some cool formatting features for a couple languages

dgalbraith33 commented on src/main/webapp/WEB-INF/view/profile.jsp:
```diff
@@ -0,0 +1,12 @@
+<body>
+
+</body>
+</html>
```
nit: Add a newline at the end of the file

eselenic committed:
Update profile.jsp

eselenic committed:
Update ProfileServlet.java

eselenic committed:
Update ProfileServletTest.java

Rafer45 committed:
Adjust whitespace a bit and arrange brackets

The brackets are now in K & R style as described in https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style

Rafer45 commented:
Looks good to me!

Rafer45 committed:
Adjust whitespace

Rafer45 commented:
Looks good to me!

dgalbraith33 reviewed (dismissed):
LGTM

eselenic committed:
Merge branch 'master' into eselenic

dgalbraith33 reviewed (approved):
LGTM
